api/v1beta1/agentserviceconfig_types.go

```diff
@@ -89,6 +89,8 @@ type AgentServiceConfigSpec struct {
 const (
 	// ConditionReconcileCompleted reports whether reconcile completed without error.
 	ConditionReconcileCompleted conditionsv1.ConditionType = "ReconcileCompleted"
+	// ConditionDeploymentsSucceeded reports whether deployments completed without error.
+	ConditionDeploymentsSucceeded conditionsv1.ConditionType = "DeploymentsSucceeded"
```

Comment:
Is it "succeeded" or "healthy"?  I think healthy, because it will become false even after the initial deployment succeeds, correct?

Reply:
Agree. `Healthy` is a better name here.